contracts/ZetaEthMock.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.7;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+
+contract ZetaEthMock is ERC20("Zeta", "ZETA") {
```

Comment:
please update it with this change 
https://github.com/zeta-chain/protocol-contracts/blob/main/contracts/evm/Zeta.eth.sol#L10

Reply:
https://github.com/zeta-chain/toolkit/blob/faet/helpers/contracts/ZetaEthMock.sol

I've updated ZetaEthMock, not sure newer commits are not showing up in this PR 😬